src/plugin/declarativedbusinterface.cpp

```diff
@@ -700,10 +700,25 @@ bool DeclarativeDBusInterface::serviceAvailable() const
     Where \c type is the D-Bus type that \c value should be marshalled as. \a arguments can be
     either a single object describing the parameter or an array of objects.
 
-    When the function returns, call \a callback with a single argument that is the return value. The
-    \a callback argument is optional, if set to \c undefined (the default), the return value will be
-    discarded. If the function fails \a errorCallback is called if it is not set to \c undefined
-    (the default).
+    A method with the D-Bus sigature \c{ssa\{sv\}}, so two string parameters and an array of dict
+    can be called like this:
+
+    \code
+    typedCall("setInventory", [
+        { "type" : 's', "value": "apple" },
+        { "type" : 's', "value": "banana"} ,
```

Comment:
Minor detail: the last space could be moved after the "banana" value so the style matches the above line.